frontend/packages/dev-console/src/utils/icons.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { BoltIcon, CatalogIcon, DatabaseIcon, LaptopCodeIcon } from '@patternfly/react-icons';
+
+import * as yamlIcon from '../images/yaml.svg';
+import * as gitIcon from '../images/from-git.svg';
+import * as dockerfileIcon from '../images/dockerfile.svg';
+import * as devfileIcon from '../images/devfile.svg';
+
+export const gitIconSVG = gitIcon;
```

Comment:
Check out https://github.com/tc39/proposal-export-ns-from#current-ecmascript-2015-modules

Can we do `export { gitIcon as gitIconSVG } from '../images/from-git.svg';` here, in order to cut down the complexity?

Reply:
SVG modules do not use named exports so we would need something like `export { * as gitIconSVG } from '../images/from-git.svg';` which is not supported as of yet. Its a proposal for ES2021 I think https://github.com/tc39/proposal-export-ns-from#proposed-addition.

Reply:
> SVG modules do not use named exports
This is the part I was missing, thanks for clarifying.

It seems that `export * from "mod";` proposal is stage 4 (already merged to ECMA-262) and TypeScript 3.8 [should support it](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#export--as-ns-syntax) so maybe we can use it already?

Reply:
Hmm, maybe webpack 4.x version we use [might not support it yet](https://github.com/webpack/webpack/issues/10460), so if it doesn't work, I'm perfectly fine with existing code.

Reply:
OK never mind, this is only supported in webpack 5.x ([link](https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.21)) so can't use it yet.